### PR TITLE
Feature/bootstrap initialization

### PR DESCRIPTION
**Pull request by Avendor7**
Bootstrap installed with basic markup in place for the application. 

Next steps are to install ember-cli-sass and convert current CSS and then break up the HTML code in application.hbs into components. 

re-run 
`npm-install`

### CODE REVIEW

File: client/app/styles/app.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+.placeholders h4 {
+  margin-bottom: 0;
+}
+.placeholder {
+  margin-bottom: 20px;
+}
+.placeholder img {
+  display: inline-block;
+  border-radius: 50%;
+}
```
Comment (Zeragamba) [inline client/app/styles/app.css:105]: open a feature request to add Sass support

File: client/app/templates/application.hbs [hunk 2]
```diff
@@ -1,22 +1,87 @@
...
+            </div>
+            {{#bs-button-group value=outcome type="radio" onChange=(action (mut outcome)) as |bg|}}
+              {{#bg.button type="default" value='W'}}Win{{/bg.button}}
+              {{#bg.button type="default" value='L'}}Loss{{/bg.button}}
+              {{#bg.button type="default" value='D'}}Draw{{/bg.button}}
+            {{/bs-button-group}}
+            <button type="submit" class="btn btn-default">Save</button>
+          </form>
+          <br>
+          <h2 class="page-header">Dashboard</h2>
```
Comment (Zeragamba) [inline client/app/templates/application.hbs]: can probably remove these placeholders

File: client/app/templates/application.hbs [hunk 3]
```diff
@@ -1,22 +1,87 @@
...
+            </div>
+            <div class="col-xs-6 col-sm-3 placeholder">
+              <img src="http://placehold.it/200x200" width="200" height="200" class="img-responsive" alt="Generic placeholder thumbnail">
+              <h4>Label</h4>
+              <span class="text-muted">Something else</span>
+            </div>
+          </div>
+          <h2 class="sub-header">Season 4</h2>
+          <div class="table-responsive">
+                <table class="table table-striped">
```
Comment (Zeragamba) [inline client/app/templates/application.hbs]: indent is off

File: client/app/templates/application.hbs [hunk 4]
```diff
@@ -1,22 +1,87 @@
...
+            <div class="form-group">
+              <label for="rank">Rank</label>
+              <input type="rank" class="form-control" id="rank" size="4" maxlength="4">
+            </div>
+            {{#bs-button-group value=outcome type="radio" onChange=(action (mut outcome)) as |bg|}}
+              {{#bg.button type="default" value='W'}}Win{{/bg.button}}
+              {{#bg.button type="default" value='L'}}Loss{{/bg.button}}
+              {{#bg.button type="default" value='D'}}Draw{{/bg.button}}
+            {{/bs-button-group}}
+            <button type="submit" class="btn btn-default">Save</button>
```
Comment (Zeragamba) [inline client/app/templates/application.hbs]: functionality will be added later?

File: client/app/templates/application.hbs [hunk 5]
```diff
@@ -1,22 +1,87 @@
...
-          <td>{{game.id}}</td>  
-          <td>{{game.createdAt}}</td> 
-          <td>{{game.rank}}</td> 
-          <td>{{game.outcome}}</td>
-        </tr> 
-        {{/each}} 
-    </tbody> 
-</table> 
+<nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container-fluid">
```
Comment (Zeragamba) [inline client/app/templates/application.hbs]: please fix indentation

File: client/app/templates/application.hbs [hunk 6]
```diff
@@ -1,22 +1,87 @@
...
+    </nav>
 
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-offset-2 col-sm-8 col-md-8 main">
+          <h2>Add New Game</h2>
+          <form class="form-inline">
+            <div class="form-group">
+              <label for="rank">Rank</label>
+              <input type="rank" class="form-control" id="rank" size="4" maxlength="4">
```
Comment (Zeragamba) [inline client/app/templates/application.hbs]: input type "rank" doesn't exist. should be "number"?

**Review by Zeragamba — CHANGES_REQUESTED** [review]

File: client/app/templates/application.hbs (hunk 6 above)
Comment (Avendor7) [reply client/app/templates/application.hbs]: good catch

File: client/app/templates/application.hbs (hunk 4 above)
Comment (Avendor7) [reply client/app/templates/application.hbs]: yes

File: client/app/templates/application.hbs (hunk 2 above)
Comment (Avendor7) [reply client/app/templates/application.hbs]: they will get moved to a component in the next feature when I move everything to components - we discussed this. I don't want to delete the code right away when we will need it in the future.

File: client/app/templates/application.hbs (hunk 3 above)
Comment (Avendor7) [reply client/app/templates/application.hbs]: fixed

File: client/app/templates/application.hbs (hunk 5 above)
Comment (Avendor7) [reply client/app/templates/application.hbs]: fixed

**Review by Zeragamba — APPROVED** [review]

**Pull request merged**